### ISSUE TITLE
Help for my acr repository list not formatted correctly

### ISSUE THREAD

**Issue by spottedmahn** [Azure/azure-cli#10676]
## Describe the bug
Isn't the following missing a CRLF?

![image](https://user-images.githubusercontent.com/14079228/65793318-85730480-e133-11e9-83de-a9df212d936f.png)

**Command Name**
`az acr repository list`

**Errors:**
n/a

## To Reproduce:
- `az acr repository list`

## Expected Behavior
Shouldn't it be like this?

![image](https://user-images.githubusercontent.com/14079228/65793703-717bd280-e134-11e9-8a37-bac31570ce51.png)

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.6
Shell: powershell.exe

azure-cli 2.0.74

```

**Comment by yugangw-msft** [comment]
This is not an ACR specific issue rather Python ArgParse's default behavior. @jiasli , let us keep it active in case we can do something at core layer to make the output layout more reasonable.

**Comment by jiasli** [comment]
[argparse](https://docs.python.org/3/library/argparse.html) doesn't print usage for every argument in a new line, according to https://github.com/python/cpython/blob/master/Lib/argparse.py#L324. It adds CRLF when `COLUMNS` env var (line width, by default 80) is reached. 

If you find `COLUMNS` to be too small, set `COLUMNS` env var so that argparse can adjust the wrapping:
- Linux: `export COLUMNS`
- Windows CMD: `set COLUMNS={}`
- PowerShell: `$Env:COLUMNS={}`

Sample output:
```
usage: az acr repository list [-h] [--verbose] [--debug] [--output {json,jsonc,table,tsv,yaml,none}]
                              [--query JMESPATH] [--subscription _SUBSCRIPTION] --name REGISTRY_NAME
                              [--top TOP] [--resource-group RESOURCE_GROUP_NAME]
                              [--suffix TENANT_SUFFIX] [--username USERNAME] [--password PASSWORD]
```

In Python 3.8, argparse will automatically get the terminal size using [shutil.get_terminal_size](https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size): https://github.com/python/cpython/blob/3.8/Lib/argparse.py#L169

**Comment by spottedmahn** [comment]
Thanks for the quick response.  I see now, there's a dependency on [argparse](https://docs.python.org/3/library/argparse.html) that controls the behavior.

My 2cents is I don't necessarily care that every arg is on its own line but that **an arg doesn't cross** lines, I find that hard to follow.

**Comment by jiasli** [comment]
`{}` should be replaced with a number.
